podcast/2.6.md

```diff
@@ -0,0 +1,741 @@
+
+**Orta Therox:**
+Yep.
+
+**Beyang Liu:**
+Before CocoaPods there's like nothing.
+
+**Orta Therox:**
+And before the ecosystem vendor, Apple, just said that's not a problem that needs to be solved. You shouldn't be sharing code, was actually their opinion, and thus CocoaPods was reactionary even to that, the idea that open source could exist was a push in that ecosystem at the time. It's much less now, obviously.
+
+**Beyang Liu:**
+How did you go from Mac and iOS native development into the JS and TS and web world?
+
+**Orta Therox:**
+Pretty smooth transition. I was forced into it.
+
+**Beyang Liu:**
+How so?
+
+**Orta Therox:**
+Well, React Native is such a compelling use case in comparison to native code. I had wrote native code for about 10 years when React Native come along, and I knew all of the flaws of native code that can also be seen as positives, but also iteration times.
+
+**Beyang Liu:**
+What were the big trade-offs in your mind?
+
+**Orta Therox:**
+Native code tends to be more complicated, because systems tend to be more separated. Okay, to give more specific examples. Think about it as if your website required you to code sign one part differently to another part, if your mobile version of your app had to not be sharing a domain, but it had to be a completely separate app bundle that would be included, but may also have different code signing rules, and all of those... each one of those also needs to be compiled in specific ways. Those are reasonable problems that you solve once or twice a month. But the hard one is just iteration times are absolutely terrible, especially if you're targeting for a device that's not the device you're working on. Then suddenly you have to build for that device, send it over, restart whatever simulator or emulator you've got going, then you're up and running, and you lose all state and things like that.
+
+Apple definitely took about a decade to get on board with trying to improve that iteration cycle, but realistically, even their current approach, which is SwiftUI, it doesn't actually run your code and hot-reload it. It actually runs it separately in a sandbox that isn't your application, so it's not actually that accurate. It's just close enough. And so, you still don't get the React style. I press save and it does it all.
+
+**Beyang Liu:**
+Yeah. It's magical.
+
+**Orta Therox:**
+All those things were such compelling cases to React Native. There was just so much React advantage, and then there was also we got the advantage of using Relay, which is a really powerful way of connecting your API to your client. And we said that the apps I was working on, at Artsy, these were apps that were just pretty JSON parses, and native development SDKs, these are really built for building long-running applications like web browsers or your music player, messaging apps, not these quick, like, "Get me some JSON data, put it on a screen and then don't do anything until you click something next."
```

Comment:
```suggestion
All those things were such compelling cases to React Native. There was just so much React advantage, and then there was also we got the advantage of using Relay, which is a really powerful way of connecting your API to your client. And we said that the apps I was working on, at Artsy, these were apps that were just pretty JSON parsers, and native development SDKs, these are really built for building long-running applications like web browsers or your music player, messaging apps, not these quick, like, "Get me some JSON data, put it on a screen and then don't do anything until you click something next."
```
@csbailey5t I didn't want to just change this, but was this a typo?

Reply:
Yes, I think it's a typo after listening again.